vcr/issuer/issuer.go

```diff
@@ -133,6 +133,10 @@ func (i issuer) Issue(ctx context.Context, template vc.VerifiableCredential, opt
 		return nil, fmt.Errorf("unable to store the issued credential: %w", err)
 	}
 
+	if err = i.vcrStore.StoreCredential(*createdVC, nil); err != nil {
```

Comment:
there's a load API for this. Not needed.

Reply:
Agree, will go